src/main/kotlin/net/atos/zac/mail/MailService.kt

```diff
@@ -74,14 +81,32 @@ class MailService @Inject constructor(
         lateinit var mailSession: Session
 
         // http://www.faqs.org/rfcs/rfc2822.html
-        private const val SUBJECT_MAXWIDTH = 78
+        private const val SUBJECT_MAX_WIDTH = 78
 
         private const val FONT_SIZE = 16f
         private const val MAIL_VERZENDER = "Afzender"
         private const val MAIL_ONTVANGER = "Ontvanger"
         private const val MAIL_BIJLAGE = "Bijlage"
         private const val MAIL_ONDERWERP = "Onderwerp"
         private const val MAIL_BERICHT = "Bericht"
+
```

Comment:
Good to add a comment that this is a Java Mail variable? Was not clear to me initially. https://javaee.github.io/javamail/docs/api/com/sun/mail/smtp/package-summary.html

Reply:
Added the above link and changed constant name